packages/datatrak-web/src/features/Survey/Components/SurveyQuestionGroup.tsx

```diff
@@ -56,7 +56,7 @@ export const SurveyQuestionGroup = ({ questions }: { questions: SurveyScreenComp
                 detailLabel={detailLabel}
                 id={questionId}
                 code={questionCode}
-                name={questionId}
+                name={questionCode}
```

Comment:
I'm not 100% sure if it's better to use the questionId, or the questionCode for the form data. It might make more sense one way or the other when we do the validation and form submit later on